src/resources/startup.py

```diff
@@ -305,7 +305,7 @@ def Execute(self, request, context):
                 output_shape = output_np_array.shape
 
                 # We need to serialize TYPE_STRING
-                if output_np_array.dtype.type is np.object or output_np_array.dtype.type is np.bytes_:
+                if output_np_array.dtype.type == np.object_ or output_np_array.dtype.type is np.bytes_:
```

Comment:
You still need to remove `type` from `output_np_array.dtype.type`.

Reply:
Got it. Now it should be consistent with line 79.